server/src/test/java/io/crate/execution/ddl/tables/AddColumnTaskTest.java

```diff
@@ -258,40 +256,6 @@ public void test_raises_error_if_column_already_exists_with_different_type() thr
         }
     }
 
-    @Test
-    public void test_raises_error_on_nested_arrays() throws Exception {
-        var e = SQLExecutor.builder(clusterService)
-            .addTable("create table tbl (x int)")
-            .build();
-        DocTableInfo tbl = e.resolveTableInfo("tbl");
-        ClusterState state = clusterService.state();
-        try (IndexEnv indexEnv = new IndexEnv(
-            THREAD_POOL,
-            tbl,
-            state,
-            Version.CURRENT
-        )) {
-            var addColumnTask = new AddColumnTask(e.nodeCtx, imd -> indexEnv.mapperService());
-            SimpleReference newColumn1 = new SimpleReference(
-                new ReferenceIdent(tbl.ident(), "y"),
-                RowGranularity.DOC,
-                new ArrayType<>(new ArrayType<>(DataTypes.LONG)),
-                2,
-                null
-            );
-            List<Reference> columns = List.of(newColumn1);
-            var request = new AddColumnRequest(
-                tbl.ident(),
-                columns,
-                Map.of(),
-                new IntArrayList()
-            );
-            assertThatThrownBy(() -> addColumnTask.execute(state, request))
-                .isExactlyInstanceOf(MapperParsingException.class)
-                .hasMessageContaining("nested arrays are not supported");
-        }
```

Comment:
Maybe keep the test but change this to assert that the column in the new DocTableInfo has the expected type?